src/views/continued-illness.njk

```diff
@@ -0,0 +1,64 @@
+{% extends 'layout.njk' %}
+
+{% from 'govuk/components/back-link/macro.njk'      import govukBackLink %}
+{% from 'govuk/components/error-summary/macro.njk'  import govukErrorSummary %}
+{% from 'govuk/components/button/macro.njk'         import govukButton %}
+{% from 'govuk/components/radios/macro.njk'         import govukRadios %}
+
+{% block pageTitle %}
+  Appeal due to ill health: Is the person still ill?
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {{
+        govukErrorSummary ({
+          titleText: 'There is a problem with the details you gave us',
+          errorList: validationResult.errors
+        }) if validationResult and validationResult.errors.length > 0
+      }}
+
+      <form method="post" id="continued-illness-form">
```

Comment:
I wonder whether we need form ID. Why did you add it?

Reply:
Good question. I thought I'd followed the GOV.UK Design System example but apparently not - this is an artefact of referencing the Select The Penalty page. I'll remove.

Reply:
I generally do not like having things that are useless cause their existence might suggest otherwise.

Reply:
Absolutely.